requires/installation.txt

```diff
@@ -0,0 +1 @@
+tornado>=4.0,<6.1
```

Comment:
I'd prefer pinning to `<7` unless we find breakage in a minor release. Doing so would also be consistent another of other sprockets (sprockets.http, sprockets.mixins.metrics, etc).

I also realize that Tornado<5 is being kept around due to your PR only being a patch release.

Reply:
On second thought, this module is already coded using the `async` and `await` keywords. I'm pretty sure some of the earlier Tornado 4.x versions don't work correctly with those.

Reply:
http://www.tornadoweb.org/en/stable/releases/v4.3.0.html#nov-6-2015

>=4.3 is good, changing pin to reflect that.